WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -50,7 +50,10 @@
           <ul class="dropdown-menu" role="menu">
             <li><a href="/results/events.php"><%= icon('signal', class: 'fa-fw fa-rotate-90') %> <%= t '.rankings' %></a></li>
             <li><a href="/results/regions.php"><%= icon('trophy', class: 'fa-fw') %> <%= t '.records' %></a></li>
-            <li><a href="<%= persons_path %>"><%= icon('user', class: 'fa-fw') %> <%= t '.persons' %></a></li>
+            <li><a href="<%= persons_path %>"><%= icon('users', class: 'fa-fw') %> <%= t '.persons' %></a></li>
```

Comment:
This is the result with no fixed-width just on "My results":
![no-fw](https://cloud.githubusercontent.com/assets/1007485/17775981/10edcc5a-655b-11e6-9889-04c6308149a7.png)

This is the result without any `fa-fw` on navigations' icons:
![no-fw-anywhere](https://cloud.githubusercontent.com/assets/1007485/17776016/2608e728-655b-11e6-91cd-cb5318d68ea5.png)

Both look bad :(